tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,421 @@
+Use the plot "Settings" function to add new scatter plots showing the magnitude and the phase angle as a function of time, ``midPointMjdTai - 60000``.  
+This will result in the left two plots, as on the screenshot below.   Note that these quantities are not correlated with time.
+Add a new scatter plot showing the r-band magnitude as a function of phase angle (right plot), showing that the phase angle and r-band magnitude are correlated.
+
+.. figure:: /_static/portal_tut03_step04a.png
+    :name: portal_tut03_step04a
+    :width: 600
+    :alt: A screenshot of three plots showing magnitude and phase angle are not correlated with time, and that magnitude is correlated with phase angle.
+
+    Three plots demonstrating that magnitude and phase angle are correlated with each other, but not with time.
+
+4.3.  Plot the topocentric and heliocentric distances of the object as a function of time.  
+Here, use the columns in the table which you generated in Step 4.1.  
+First, delete two of the the three plots prepared in Step 4.2 by clicking on the blue ``X`` in the upper right-hand part of the plot panels to make space for new plots.  
+Then add a pair of new plots, clicking on the "plot settings."  
+In both cases, enter ``midPointMjdTai - 60000`` for X-axis, but for Y axis - enter ``topocentricDist`` for one plot, and ``heliocentricDist`` for the other.  
+After you remove the panel containing the plot made in the previous step, you will see the plots as below.  
+
+.. figure:: /_static/portal_tut03_step04b.png
+    :width: 600
+    :name: portal_tut03_step04b
+    :alt: A screenshot of two plots showing the heliocentric and topocentric distance of the trans-Neptunian object as a function of time.
+
+    Heliocentric and topocentric distance of the TNO as a function of time.  
+Note the periodic change of the topocentric distance with time resulting from the Earth's motion around the Sun - a different view of the same effect you saw in Step 3.5.  
+
+.. _DP0-3-Portal-3-Step-5:
+
+Step 5. Visualize in 3-D the trajectory of the TNO by viewing the 2-D projection of its orbit
+=========================================================================================
+
+5.1.  Navigate again to the ADQL query interface.  
+Execute the query below to extract the helio- and topocentric distances of the TNO - so you can visualize its trajectory:  
+
+.. code-block:: SQL 
+
+    SELECT heliocentricX, heliocentricY, heliocentricZ,
+    topocentricX, topocentricY, topocentricZ, ssObjectId
+    FROM dp03_catalogs_10yr.SSSource
+    WHERE ssObjectId = -735085100561880491
+
+
+5.2.  Plot the heliocentric Z distance as a function of heliocentic X distance by clicking on the "plot setings" icon and selecting ``heliocenticZ`` for y and ``heliocentricX`` for x.  
+Note that the object moves slowly in heliocentric coordinate X as well as in Y (by a comparison to, e.g., Earth's motion), covering only a few au in 10 years.  
```

Comment:
The former sentence instructs the user to plot Z vs X but this sentence discusses X vs Y. Since Y vs X is the default plot, perhaps just have step 5.2 review the default plot. Then make the next step to plot Z vs X and discuss that.

Reply:
OK, great catch.